website/addons/osfstorage/tests/test_views.py

```diff
@@ -553,3 +554,46 @@ def test_cannot_delete_root(self):
         resp = self.delete(self.root_node, expect_errors=True)
 
         assert_equal(resp.status_code, 400)
+
+class TestFileTags(StorageTestCase):
```

Comment:
Would be good to include a test adds a tag with non-ascii characters (for this reason: https://github.com/CenterForOpenScience/osf.io/pull/4358/files#r43517097).